server_src/routes/databaseIssue.js

```diff
@@ -1,7 +1,159 @@
 const express = require("express");
+const path = require("path");
+const { execSync, exec } = require("child_process");
+const mongoose = require("mongoose");
+const Logger = require("../../server_src/lib/logger.js");
+const ServerSettingsDB = require("../models/ServerSettings");
+const isDocker = require("is-docker");
+const envUtils = require("../utils/env.utils");
 
 const router = express.Router();
+const logger = new Logger("OctoFarm-Server");
+
+function validateMongoURL(mongoURL) {
+  const mongoString = mongoURL.toLowerCase();
+  const hasMongoPrefix = mongoString.toLowerCase().includes("mongodb://") || mongoString.toLowerCase().includes("mongodb+srv://");
+  const hasOctoFarmTable = mongoString.includes("/octofarm");
+
+  return {
+    hasMongoPrefix,
+    hasOctoFarmTable,
+    isValid: hasOctoFarmTable || hasMongoPrefix,
+  };
+}
+
 router.get("/", (req, res) =>
```

Comment:
All the endpoints necessary for updating. Thoroughly tested, but please verify the API and ejs yourself as well!